arlo-client/src/components/AuditForms/EstimateSampleSize.test.tsx

```diff
@@ -670,4 +676,43 @@ describe('EstimateSampleSize', () => {
       expect(updateAuditMock).toBeCalledTimes(0)
     })
   })
+
+  it('handles errors from the server side', async () => {
+    apiMock.mockReset()
+    // apiMock.mockImplementation(() =>
```

Comment:
Is this code supposed to be commented out? It's not really clear just looking at the diff what expectations in this test are expecting an error.